mrrc/config.py

```diff
@@ -1,89 +0,0 @@
-from typing import List
-from configparser import ConfigParser, NoSectionError
-from mrrc.utils.logs import DEFAULT_LOGGER 
-import os
-import sys
-import logging
-import json
-
-CONFIG_FILE = "mrrc-uploader.conf"
-
-SECTION_MRRC = 'mrrc'
-MRRC_IGNORE_PATTERN = "ignore_patterns"
-
-SECTION_AWS = 'aws'
-AWS_KEY_ID = 'access_key_id'
-AWS_KEY = 'secret_access_key'
-AWS_ENDPOINT = 'endpoint_url'
-AWS_REGION = 'region'
-AWS_RETRY_MAX = 'retry_max_attempts'
-AWS_RETRY_MODE = 'retry_mode'
-AWS_BUCKET = "bucket"
-AWS_DEFAULT_BUCKET = "mrrc"
-
-
-AWS_DEFAULT_BUCKET="mrrc"
-
-logger = logging.getLogger(DEFAULT_LOGGER)
-
-class MrrcConfig(object):
```

Comment:
We still need this config for [mrrc] configurations, like ignore_patterns or something more.

Reply:
I'll suggest we shift to either command-line arguments or ENV variables for those settings also since that's the standard practice. It's very easy to access/set ENV variables compared to loading and parsing files.